addons/ai/stringtable.xml

```diff
@@ -65,5 +81,11 @@
             <Czech>Nenalezena žádná budova.</Czech>
             <Turkish>Bir yapı bulunamadı</Turkish>
         </Key>
+        <Key ID="STR_ACE_AI_AssignNVG_DisplayName">
+            <English>Assign NVG</English>
```

Comment:
The only clarification I can think of:

```suggestion
            <English>Assign NVG automatically</English>
```

Reply:
There's a difference in meaning between the plain English word assign (to allocate a thing to an entity) and the Arma assign command (to put a specific item in a valid slot).

To me, having asign in the name, while techinically correct from an Arma perspective, sounds like it's going to give NVGs to units that don't have them (which is the reason for my mistaken comment higher up), when it's actually just going to move them from inventory to NVG slot.

I think the only truely unambiguous English words might be don (put on [an item of clothing]) and doff (remove [an item of clothing]).

`AI Don and Doff NVGs Automatically` doesn't have a great ring to it though...

Reply:
Yeah, I had the same issue. Maybe adding "Does NOT give NVGs to AI!" to the description would help?

Reply:
'Equip', maybe?

Reply:
> 'Equip', maybe?

Beat me to it by 5mins. +1.

Reply:
Equip can also be understood as both.

Reply:
'Put on'

Reply:
'Make AI wear/take off NVGs automatically'